eclass-writing/text.xml

```diff
@@ -27,15 +27,16 @@ Roughly speaking, there are three kinds of eclass:
 <ul>
   <li>
     Those which provide common functions which are used by many ebuilds (for
```

Comment:
Shouldn't that be 'these that'? Maybe sort them too, the order looks arbitrary.

Reply:
I haven't changed the wording. It is the text from 2005, which :smile: was written by a native speaker of English.

They are currently sorted by popularity. I can make it alphabetical if you prefer.

Reply:
Nope, this sounds correct